pandas/tseries/timedeltas.py

```diff
@@ -21,7 +24,10 @@ def to_timedelta(arg, unit='ns', box=True, coerce=False):
     box : boolean, default True
         If True returns a Timedelta/TimedeltaIndex of the results
         if False returns a np.timedelta64 or ndarray of values of dtype timedelta64[ns]
-    coerce : force errors to NaT (False by default)
+    errors : {'ignore', 'raise', 'coerce'}, default 'raise'
+        If 'raise', then invalid parsing will raise an exception
+        If 'coerce', then invalid parsing will be set as NaT
+        If 'ignore', then invalid parsing will return the input
```

Comment:
can you add `-` markers for making this a list? (will be clearer in the html version)